src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -3874,4 +3920,31 @@ int IDebugSettingsCallback110.ShouldSuppressImplicitToStringCalls(out int pfSupp
             }
         }
     }
+
+    internal class SendInvalidateRequest : DebugRequestWithResponse<SendInvalidateArguments, SendInvalidateResponse>
```

Comment:
In your extension, are you checking the contents of this response?

If you are, could you modify it to be similar to https://github.com/microsoft/VSDebugAdapterHost/blob/3bcc0b01df7e604227e1a0334a4160777901c2ea/src/sample/SampleDebugAdapter.VSIX/CustomProtocolExtension/SampleExtension.cs#L84-L97

E.g.
```csharp
internal class SendInvalidateResponse : ResponseBody
{
        public InvalidatedAreas Areas { get; set; }
        public int? ThreadId { get; set; }
        public int? StackFrameId { get; set; }
}
```

If not, then we could remove `SendInvalidateResponse` and just use `DebugRequest`
```suggestion
    internal class SendInvalidateRequest : DebugRequest<SendInvalidateArguments>
```

Reply:
In my case, response does not play any role, I have removed all mentions